app/src/store/user.ts

```diff
@@ -3,10 +3,18 @@ import config from '../../config';
 import Auth0 from 'react-native-auth0';
 import { State, Dispatch, type AsyncStoreSlice } from './app';
 
-export type UserState = AsyncStoreSlice<{
-  accessToken: string;
-  email: string;
-}>;
+export type UserState = AsyncStoreSlice<
+  {
+    status: 'fulfilled';
```

Comment:
Sorry, I know you're probably sick of having to change things.

The types we pass to `AsyncStoreSlice` don't need `status` defined on them since they get merged by the helper type itself. So if you do something like `AsyncStoreSlice<T>`, then `T` gets merged automatically with `{ status: 'fulfilled' }`.